### ISSUE TITLE
Master does not pass tls tests.

### ISSUE THREAD

**Issue by aiverson** [luvit/luvit#1037]
Running `make test` on a clean unmodified master does not pass the tests. Is this functionality that hasn't been developed yet, or a regression?

```
# Starting Test: tls-server-verify - Do not request certs. Everyone is unauthorized.
    connecting with agent1
'openssls_client -tls1 -connect 127.0.0.1:32000 -key /home/aiverson/projects/upstream-luvit/tests/fixtures/keys/agent1-key.pem -cert /home/aiverson/projects/upstream-luvit/tests/fixtures/keys/agent1-cert.pem'
Uncaught Error: ...projects/upstream-luvit/tests/test-tls-server-verify.lua:111: agent1 rejected, but should NOT have been
stack traceback:
        [C]: in function 'assert'
        ...projects/upstream-luvit/tests/test-tls-server-verify.lua:111: in function 'handler'
        /home/aiverson/projects/upstream-luvit/deps/core.lua:248: in function 'emit'
        ...e/aiverson/projects/upstream-luvit/deps/childprocess.lua:128: in function <...e/aiverson/projects/upstream-luvit/deps/childprocess.lua:125>
        [C]: in function 'run'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:79: in function </home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:64>
        [C]: in function 'xpcall'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:64: in function 'run'
        /home/aiverson/projects/upstream-luvit/tests/libs/tap.lua:165: in function 'tap'
        /home/aiverson/projects/upstream-luvit/tests/run.lua:42: in function 'fn'
        [string "bundle:deps/require.lua"]:310: in function 'require'
        /home/aiverson/projects/upstream-luvit/main.lua:118: in function 'main'
        /home/aiverson/projects/upstream-luvit/init.lua:49: in function </home/aiverson/projects/upstream-luvit/init.lua:47>
        [C]: in function 'xpcall'
        /home/aiverson/projects/upstream-luvit/init.lua:47: in function 'fn'
        [string "bundle:deps/require.lua"]:310: in function <[string "bundle:deps/require.lua"]:266>
make: *** [Makefile:12: test] Error 255
```

**Comment by Miigon** [comment]
It's likely that you are using an obsolete version of openssl.
The bug is fixed with commit fad9324c730a0ac5dcd3878af3e1dce727153e7a. Please try the newest version again and see if the problem is solved or not.

**Comment by KelsonBall** [comment]
I'm also having this issue, including after upgrading openssl and libssl-dev packages
